app/controllers/api/v1/accounts/portals_controller.rb

```diff
@@ -2,7 +2,7 @@ class Api::V1::Accounts::PortalsController < Api::V1::Accounts::BaseController
   before_action :fetch_portal, except: [:index, :create]
 
   def index
-    @portals = Current.account.portals
+    @portals = Current.account.portals.active
```

Comment:
In the internal APIs, we should list all the portals, right?

Reply:
Yes, I forgot to remove this after our discussion yesterday. 👍🏼